### DISCUSSION TITLE
Is authentik a "One man show"?

### DISCUSSION THREAD

**Discussion by guettli** [goauthentik/authentik#3949] [General]
I looked at the commit since January 1, 2022.

![image](https://user-images.githubusercontent.com/414336/200302957-14252023-ed57-4823-b723-53e81c2bedc2.png)

Source: https://github.com/goauthentik/authentik/graphs/contributors?from=2022-01-01&to=2022-11-07&type=c

Almost all commits are from BerryJu.

Before settling on a software I check the difference between the foure people making the most commits.

the numbers of commits are: 1070, 6, 4, 3.

Or am I missing something and above link does not show the correct distribution?

What happens if BerryJu gets hit by bus?

**Comment by fbartels** [comment]
This may be relevant to you @guettli https://goauthentik.io/blog/2022-11-02-the-next-step-for-authentik

**Reply by guettli** [reply]
@fbartels So Authentik will have a great future. Thank you for the good news.
